### ISSUE TITLE
The current build seems broken with the new Paperclip and PDF gems.

### ISSUE THREAD

**Issue by buckspencer** [tiy-austin-ror-jan2015/tiy-rails-template#29]

**Comment by jah2488** [comment]
How so @buckspencer? Is there an error associated with it? What part isn't working? Can you show me what code you have that isn't working?

_(Submitting a bug report normally entails providing a decent amount of documentation that demonstrates what you were doing and how it is not working as expected)_

**Comment by buckspencer** [comment]
Im currently getting this error. 
`open_http': 406 Not Acceptable (OpenURI::HTTPError)
when running the generator

**Comment by buckspencer** [comment]
Has the current iteration of the generator been tested?

**Comment by jah2488** [comment]
Good question. Might need to ask Jacob. 

—
Justin Herrick

**Comment by buckspencer** [comment]
We should probably also add a readme so people know what they are getting and running.

**Comment by Jmingus** [comment]
So the generator just stops with error? or is it when you try to open up localhost?

**Comment by jah2488** [comment]
@buckspencer Good idea

@Jmingus Good point.

**Comment by jah2488** [comment]
I just tested the generator on my machine and didn't get that error. I did have to run `rake db:migrate` after it was done, because of active admin, but other than that its been fine.

**Comment by buckspencer** [comment]
Are there any assumptions the generator makes as to what gems I need to run the generator.

**Comment by Jmingus** [comment]
I also don't have the error, also what should be in the readme, just like what gems it's installing?

**Comment by Jmingus** [comment]
Well you need rails and ruby, but otherwise no I don't think so

**Comment by jah2488** [comment]
@buckspencer Nope, if you can run `rails new` you should be able to run the template generator. The error you posted seemed like you might not have had internet access at the time when the generator was trying to install a gem or some other file.

@Jmingus I created an issue with what I feel would suffice for a readme. It looks like Buck is taking care of that too :+1:

**Comment by buckspencer** [comment]
Yeah I am writing the readme.  My internet IS really spotty ATM. My apartment complex is trying to make life hell for me RN.

**Comment by buckspencer** [comment]
Ill run the gen again to check.

**Comment by buckspencer** [comment]
Are you guys typing in -T still before generating?

**Comment by Jmingus** [comment]
What does -T do again?

**Comment by Jmingus** [comment]
Gets rid of tests?

**Comment by buckspencer** [comment]
yeh
